x-pack/platform/plugins/shared/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/analytics_detail_flyout/analytics_detail_flyout.tsx

```diff
@@ -103,8 +103,8 @@ export const AnalyticsDetailFlyout = () => {
           <EuiFlexItem grow={false}>
             <EuiButtonEmpty onClick={redirectToAnalyticsList}>
               <FormattedMessage
-                id="xpack.ml.jobDetailsFlyout.openJobsListButton"
-                defaultMessage="Open jobs list"
+                id="xpack.ml.overview.analyticsList.manageJobsButtonText"
+                defaultMessage="Manage jobs"
```

Comment:
This flyout only ever shows 1 job doesn't it? In which case, since we are passing a filter to only show the selected job in the Jobs list, I think we should drop the `s` to be `Manage job`.